.github/workflows/ci.yml

```diff
@@ -5,12 +5,22 @@ on: [push, pull_request]
 jobs:
   tests:
     runs-on: ubuntu-20.04
+    container:
+      image: "ubuntu:20.04"
     steps:
       - uses: actions/checkout@v2
       - name: Install required dependencies
         run: |
-          sudo apt install --yes npm
+          apt update
+          apt install -y sudo
+          sudo apt install -y git
+          sudo apt install -y gcc g++ make
```

Comment:
@parhamsaremi why are gcc and g++ and make needed?

Reply:
They were in the steps of installing nodejs :thinking:. I can remove them and check if there'll be a problem. :thinking:

Reply:
I tested it in this branch: https://github.com/parhamsaremi/conventions/commits/vanillaCI-test-gcc

I guess it's not needed. I'll remove it now.